### DISCUSSION TITLE
AI作曲原來這麼好聽

### DISCUSSION THREAD

**Discussion by 110302001** [1101-GS4719-C/Python-Community-GS4719-C#1568] [閒聊八卦]
之前在YouTube 上看到了這個影片，覺得ai 做的比想像中好聽很多,又想到之前去觀賞了學校辦的音樂比賽，其中有一個項目就是創作組，就覺得我好像很難分出一首曲子好或壞，可能覺得一首歌旋律好很洗腦就會很喜歡。更
加覺得的現在的流行音樂很有機會Ai取代
尤其是抖音歌那種但那種要求背景和場景情感抒發要求的原聲帶應該還是比較難取代,不過作曲家和Ai一起工作作曲是趨勢。畢竟Ai目前還無法理解人類的情感,看看會不會有一天就被突破了
[https://youtu.be/bYyHESOjqJw]

**Comment by ThanatosJun** [comment]
如果你想實做關於AI 作曲或歌歌，我提供以下兩段影片，我滿喜歡他們的，可以說在剛接觸SD後Youtube就推播他們給我，雖然我還沒踏入語音的 AI 生成領域！！
[現下火紅的AI孫燕姿怎麼做?幾步驟輕鬆達成!!RVC基礎操作教學](https://www.youtube.com/watch?v=H83xhjIbtZk)
[超簡單RVC標準安裝，讓你成為RVC使用者的先驅?!](https://www.youtube.com/watch?v=4MuYTBeMmFc)

**Comment by BOSENCHANG2004** [comment]
我覺得AI作曲一開始聽著到新奇但是越聽會越覺得其實每首歌曲並沒有太大的變化，或許受限於訓練AI的資料，AI創作出來的因為缺乏多樣性，所以我認同AI作曲目前依然無法取代真人作曲。
